rqd/rqd/rqconstants.py

```diff
@@ -66,6 +66,7 @@
 RQD_RETRY_CRITICAL_REPORT_DELAY = 30
 RQD_USE_IP_AS_HOSTNAME = True
 RQD_USE_IPV6_AS_HOSTNAME = False
+RQD_USE_PATH_ENV_VAR = False
```

Comment:
Please add a comment describing what this new setting does. Currently it's hard to understand all these settings, ideally over time we will add more comments to explain all of them, but at least adding comments for any new setting will help things from getting worse :)

Reply:
Done!